### PR TITLE
Formatting Bug Fixes

### PR DESCRIPTION
**Pull request by tarik-brown**

### CODE REVIEW

File: Extension/package.json [hunk 1]
```diff
@@ -114,11 +114,11 @@
         "C_Cpp.vcFormat.indent.multiLineRelativeTo": {
           "type": "string",
           "enum": [
-            "outermost_parenthesis",
-            "innermost_parenthesis",
-            "statement_begin"
+            "outermostPparenthesis",
```
Comment (bobbrow) [inline Extension/package.json]: Remove the extra 'p'

File: Extension/package.json [hunk 2]
```diff
@@ -114,11 +114,11 @@
...
+            "innermostPparenthesis",
```
Comment (bobbrow) [inline Extension/package.json]: here too. :)

File: Extension/package.json [hunk 3]
```diff
@@ -195,30 +195,30 @@
...
           "type": "string",
           "enum": [
-            "new_line",
-            "same_line",
+            "newLine",
+            "sameLine",
             "ignore"
           ],
-          "default": "ignore",
+          "default": "Ignore",
```
Comment (bobbrow) [inline Extension/package.json]: It should be a lowercase 'i' here

File: Extension/package.json [hunk 4]
```diff
@@ -195,30 +195,30 @@
...
```
Comment (bobbrow) [inline Extension/package.json]: it should be a lowercase 'i' here

File: Extension/src/LanguageServer/client.ts [hunk 5]
```diff
@@ -1573,8 +1573,11 @@ export class DefaultClient implements Client {
             C_Cpp: {
                 ...cppSettingsScoped,
                 vcFormat: {
+                    // Reach all settings
+                    ...vscode.workspace.getConfiguration("C_Cpp.vcFormat", this.RootUri),
```
Comment (bobbrow) [inline Extension/src/LanguageServer/client.ts:1575]: Are there any settings at this level that you need to get?

File: Extension/package.json [hunk 6]
```diff
@@ -698,7 +697,7 @@
             ]
           },
           "description": "%c_cpp.configuration.commentContinuationPatterns.description%",
-          "scope": "resource"
+          "scope": "window"
```
Comment (bobbrow) [inline Extension/package.json]: I'm not sure why these are showing up in your branch.  I thought @Colengms did that in another PR.

File: Extension/src/LanguageServer/client.ts [hunk 7]
```diff
@@ -389,7 +389,7 @@ const GetDocumentSymbolRequest: RequestType<GetDocumentSymbolRequestParams, Loca
 const GetSymbolInfoRequest: RequestType<WorkspaceSymbolParams, LocalizeSymbolInformation[], void, void> = new RequestType<WorkspaceSymbolParams, LocalizeSymbolInformation[], void, void>('cpptools/getWorkspaceSymbols');
 const GetFoldingRangesRequest: RequestType<GetFoldingRangesParams, GetFoldingRangesResult, void, void> = new RequestType<GetFoldingRangesParams, GetFoldingRangesResult, void, void>('cpptools/getFoldingRanges');
 const GetSemanticTokensRequest: RequestType<GetSemanticTokensParams, GetSemanticTokensResult, void, void> = new RequestType<GetSemanticTokensParams, GetSemanticTokensResult, void, void>('cpptools/getSemanticTokens');
-
+// const DocumentFormatRequest: RequestType<EditorConfigParams, , void, void> = new RequestType<GetFoldingRangesParams, GetFoldingRangesResult, void, void>('cpptools/documentFormat');
```
Comment (Colengms) [inline Extension/src/LanguageServer/client.ts]: Commented out code

File: Extension/src/LanguageServer/client.ts [hunk 8]
```diff
@@ -1007,7 +1007,7 @@ export class DefaultClient implements Client {
                             });
                         }
                     }
-
+
```
Comment (Colengms) [inline Extension/src/LanguageServer/client.ts]: This trailing spaces are causing a linter error due to no-trailing-spaces rule.

**Review by Colengms — CHANGES_REQUESTED** [review]

File: Extension/package.json (hunk 6 above)
Comment (Colengms) [reply Extension/package.json]: This got cleared by my merging with master.

File: Extension/src/LanguageServer/client.ts (hunk 5 above)
Comment (bobbrow) [reply Extension/src/LanguageServer/client.ts:1575]: In other words, can this line be deleted?

**Review by bobbrow — APPROVED** [review]

**Review by Colengms — APPROVED** [review]

**Pull request merged**